### DISCUSSION TITLE
Question - Configuring the Oauth-Proxy Image

### DISCUSSION THREAD

**Discussion by raffamendes** [jaegertracing/jaeger#3531] [Q&A]
Hi All,

I'm deploying Jaeger using the Redhat Openshift Distributed Tracing Platform on an Openshift 4.8.29 cluster, the problem is, the operator is deploying the latest version of the oauth-proxy image, with contains a reference to a configmap that only exists on Openshift 4.9 (oauth-serving-cert), is there a way to configure in the CRD which image should be pulled for the oauth-proxy??? I fixed this by manually changing the deployment object and putting the v4.8.0 tag, that works.

```go
oauthServerCert, err := p.configMapLister.ConfigMaps("openshift-config-managed").Get("oauth-serving-cert")
	if err != nil {
		return nil, err
	}
```
this is the code, that I'm referring to, and when deploying this to OCP 4.8, this is the error on the screen:
![image](https://user-images.githubusercontent.com/54948088/154117358-bc584c9c-c6e2-44d3-8213-d253242af47d.png)

**Comment by yurishkuro** [comment]
Please address this question to RedHat support, it is not related to Jaeger.

**Comment by raffamendes** [comment]
Hi @yurishkuro I Disagree, since I'm using the Jaeger CRD, and my question is about the Jaeger CRD, how it's not related to Jaeger?

**Reply by yurishkuro** [reply]
If I understood your question, you are using CRD provided by 3rd party (RedHat), not by the maintainers of this project.
